src/cli.rs

```diff
@@ -11,7 +11,11 @@ impl Cli {
     fn default_year() -> i32 {
         let now = Local::now();
         let year = now.year();
-        return if now.month() <= 3 { year - 1 } else { year };
+        if now.month() <= 3 {
```

Comment:
Wait this could be inlined with a silly trick `year - (now.month() <= 3) as i32`